nautobot/ipam/models.py

```diff
@@ -611,7 +604,7 @@ def get_child_prefixes(self):
         Return all Prefixes within this Prefix and VRF. If this Prefix is a container in the global table, return child
         Prefixes belonging to any VRF.
```

Comment:
This is definitely a weird corner case, looking forward to changing this when we have proper prefix parentage...